app/data/models/User.py

```diff
@@ -40,17 +48,18 @@ def validatePassword(self):
         # min 8 characters, max 20
```

Comment:
do not validate for password max length, just cut it to 20 if someone uses longer than 20

Reply:
but should I inform a user about it?

Reply:
no need